.env.example

```diff
@@ -0,0 +1,9 @@
+# Wallet
+PRIVATE_KEY=0x-your-private-key
+
+# API endpoints
+KAIROS_RPC_URL=https://public-en-kairos.node.kaia.io
+
+# Faucet settings
+KAIROS_FAUCET_AMOUNT=1
+RECEIVER_ADDRESS=0xe45CEA5135167451e16175f7B58E9912CF1d8b63 
```

Comment:
@avdheshcharjan what is this receiver_address for. i dont think we need this. this receiver is provided by the user itself.